src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
If I get a vote in this, I would vote for semantically equivalent serialisation.  CellML is for storage and exchange, we just need to the tools to get the same meaning from the same document, for this we only need semantically equivalent serialisation.  

Taking care of whitespace changes .... Urghhhhhh.